From ac36cd6d4aaed68627d8bb404fb8ab0caf3b55c9 Mon Sep 17 00:00:00 2001
From: kszaq <kszaquitto@gmail.com>
Date: Thu, 7 Jul 2016 00:34:59 +0200
Subject: [PATCH] drivers/amlogic/display/vout: don't re-set user-defined
 framerate after video stop

Pausing video in Kodi triggers frame rate change to pre-playback rate and then
back to automatic frame rate with fr_policy set to 2. This makes kernel not switch
the framerate back.
---
 drivers/amlogic/display/vout/tv_vout.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/amlogic/display/vout/tv_vout.c b/drivers/amlogic/display/vout/tv_vout.c
index 16fdce2..9920613 100644
--- a/drivers/amlogic/display/vout/tv_vout.c
+++ b/drivers/amlogic/display/vout/tv_vout.c
@@ -1088,7 +1088,7 @@ static int tv_set_vframe_rate_end_hint(void)
 		fps_playing_flag = 0;
 		if (DOWN_HPLL == fine_tune_mode)
 			fine_tune_mode = UP_HPLL;
-		framerate_automation_set_mode(mode_by_user, END_HINT);
+		// framerate_automation_set_mode(mode_by_user, END_HINT);
 		fine_tune_mode = KEEP_HPLL;
 		fps_target_mode = VMODE_INIT_NULL;
 		mode_by_user = VMODE_INIT_NULL;
-- 
1.8.3.1
